src/cache.cpp

```diff
@@ -9,10 +9,9 @@
 #include <cosim/fs_portability.hpp>
 #include <cosim/log/logger.hpp>
 
-#include <cstdlib>
```

Comment:
Unless this was done automatically by clang-format, I see no need to move the `#include` directive. `<cstdlib>` is a standard C++ header and should be grouped with the others.